doc/observability-deployment-rhocp4.2-4.4.adoc

```diff
@@ -6,21 +6,22 @@ The following document covers various topics for configuring and integrating you
 
 Kibana dashboards are provided for visualizing events from the Open Liberty runtime.
 
-To leverage these dashboards the logging events must be emitted in JSON format to standard-out. For information regarding how to configure an Open Liberty image with JSON logging please see link:++https://github.com/OpenLiberty/ci.docker#logging++[here].
+To leverage these dashboards the logging events must be emitted in JSON format to standard-out. For Open Liberty Operator, JSON logging is enabled by default. For information regarding how to configure an Open Liberty image with JSON logging please see link:++https://github.com/OpenLiberty/ci.docker#logging++[here].
 
 Retrieve available Kibana dashboards tuned for Open Liberty logging events link:++https://github.com/OpenLiberty/open-liberty-operator/tree/master/deploy/dashboards/logging++[here].
 
 For information regarding how to import Kibana dashboards see the official documentation link:++https://www.elastic.co/guide/en/kibana/5.6/loading-a-saved-dashboard.html++[here].
 
 For effective management of logs emitted from applications, deploy your own Elasticsearch, Fluentd and Kibana (EFK) stack. For more information see the following link:++https://kabanero.io/guides/app-logging-ocp-4-2/++[guide].
 
-Command-line JSON parsers, like JSON Query tool (jq), can be used to create human-readable views of JSON-formatted logs. In the following example, the logs are piped through grep to ensure that the message field is there before jq parses the line:
+Command-line JSON parsers, like JSON Query tool (jq), can be used to create human-readable views of JSON-formatted logs. You can create an alias for your jq command in the format you like. In the following example, the logs are piped through grep to ensure that the message field is there before jq parses the line:
 
 [source,sh]
 ----
+alias prettylog="jq .message -r"
```

Comment:
a couple of thoughts... 
1. could you put the grep into the alias as well?
2. make jq include other fields like ibm_datetime and level as well (putting a "\t" after level would help align the following field)